### DISCUSSION TITLE
Sound Level Graph?

### DISCUSSION THREAD

**Discussion by NateEaton** [stas-sl/esphome-sound-level-meter#15] [General]
Stas, 

First off, great solution. It was straightforward to throw an ESP32 and an INMP441 on a breadboard, run ESPHOME to install the advanced config and having a sound meter up and running in no time. Comparing results against a sound meter on my phone, the results are pretty close (4-5dB higher than my phone in a quiet room but peaks for loud noises are almost the same). 

I haven't gotten into lower level details, yet, messing around with filters. In the meantime, one favor? I'm curious what you used to create the example [Sound Level graph](https://user-images.githubusercontent.com/4602302/224789124-a86224c9-c11d-4972-a564-b042bab97bcb.png) in the README? It's obviously not the standard Home Assistant history graph. 

Thanks,
Nate

**Comment by stas-sl** [comment]
Hi! 

Yes, I also noticed that in a quite room I get higher readings than I would expect. Here I described my experiments: https://github.com/ikostoski/esp32-i2s-slm/issues/19. It can vary a bit (1-2 dB) depending on mic instance (I also used INMP441) and there could be other electromagnetic noises that can affect.

Considering the graph in README - I send data to my own database and use [Grafana](https://grafana.com/) for visualization.

**Comment by pe0ter** [comment]
I have built a teensybat bat detector last week and I am amazed how much high frequency noise surrounds us in our houses. Power supplies, induction cookers, anything with a switched-mode power supply creates noise a mems microphone can hear and will display as noise level even though we do not hear it and regard a room as quiet.
